wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -766,11 +766,11 @@ public final class SessionManager: NSObject, SessionManagerType {
                             return
                         }
 
+                        accountManager.select(account)
```

Comment:
This fixes an issue where in case loading the session fails we still have the previous account selected which doesn't really agree with the UI state (e.g the account switcher should only show non selected accounts, but the switcher is shown when failing to load the session).

I think it makes more sense that we select the account, then try to load the session. That way in case of a session failure, we can switch to another account.